next.config.js

```diff
@@ -387,6 +387,10 @@ const nextConfig = {
       {
         source: `/learn/pre-built-integrations/sample2`,
         destination: `/learn/integration/pre-built-integrations/sample2`,
+      },
+      {
+        source: `/learn/case-studies/wso2-vs-ballerina`,
```

Comment:
Remove this block. Case studies does not come under Learn. The url should be b.io/case-studies

Reply:
Removed.